charts/dev/livebook/questions.yaml

```diff
@@ -16,11 +16,38 @@ questions:
                                     type: dict
                                     attrs:
                                       - variable: LIVEBOOK_PASSWORD
-                                        label: "LIVEBOOK_PASSWORD"
+                                        label: "Password (LIVEBOOK_PASSWORD)"
                                         description: "Password needed to access livebook (must be at least 12 characters)"
                                         schema:
                                           type: string
                                           default: "livebookiscool"
+                                      - variable: Livebook Cookie (LIVEBOOK_COOKIE)
+                                        label: "LIVEBOOK_COOKIE"
+                                        description: "Sets the cookie for running Livebook in a cluster. Random string is generated if blank."
+                                        schema:
+                                          type: string
+                                      - variable: LIVEBOOK_SECRET_KEY_BASE
+                                        label: "Secret Key Base (LIVEBOOK_SECRET_KEY_BASE)"
+                                        description: >
+                                          Sets a secret key that is used to sign and encrypt the session and other payloads
+                                          used by Livebook. Must be at least 64 characters long and it can be generated by
+                                          commands such as: openssl rand -base64 48. Defaults to a random secret on every boot.
+                                          String is auto-generated if left blank.
+                                        schema:
+                                          type: string
+                                      - variable: LIVEBOOK_DEBUG
+                                        label: "Debug Logging (LIVEBOOK_DEBUG)"
+                                        description: >
+                                          enables verbose logging, when set to "true". Disabled by default.
+                                        schema:
+                                          type: boolean
+
+                                      - variable: LIVEBOOK_UPDATE_INSTRUCTIONS_URL
+                                        label: "Update instruction URL (LIVEBOOK_UPDATE_INSTRUCTIONS_URL)"
+                                        description: >
+                                          sets the URL to direct the user to for updating Livebook when a new version becomes available.
+                                        schema:
+                                          type: string
```

Comment:
Is this supposed to be empty by default?

Reply:
Yes.